apps/camera/js/views/controls.js

```diff
@@ -148,19 +152,22 @@ module.exports = View.extend({
   },
 
   setMode: function(mode) {
+    mode = mode || 'picture';
     debug('set mode: %s', mode);
     this.set('mode', mode);
     this.switchPosition = this.switchPositions[mode];
     var ratio = { left: 0, right: 1 }[this.switchPosition];
-    this.updateSwitchPosition();
+    this.updateSwitchPosition(mode);
```

Comment:
Perhaps a comment to say why this would ever be called without a `mode` argument.